cldr.go

```diff
@@ -1642,7 +1642,39 @@ var monthLookup = map[string]monthIndexes{
 	"zu-ZA":          {749, 749, 751, 751, 750, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
 }
 
-func fmtMonth(locale string, calendar calendarType, context, width string) func(int, string) string {
+// fmtYear formats year as numeric. The assumption is that f is "06" or "2006".
+func fmtYear(digits digits) func(y int, f string) string {
+	return func(y int, f string) string {
+		year := strconv.Itoa(y)
+
+		// ptime.Time.Format is very slow. Make it fast!
+		if f == "06" {
+			switch len(year) {
+			default:
+				year = year[len(year)-2:]
```

Comment:
year 1 will panic here

Reply:
It won't. `case 1` handles 1-digit years.